bin/ember-template-lint.js

```diff
@@ -170,9 +166,15 @@ function run() {
   }
 
   let errors = {};
-  let relativeFilePaths = fileArgs.length === 0 ? [STDIN] : expandFileGlobs(fileArgs);
+  let filesToLint;
+
+  if (positional.length === 0 || positional.includes('-') || positional.includes(STDIN)) {
+    filesToLint = new Set([STDIN]);
```

Comment:
This will strait up block any globbing. Which means the use of STDIN is an exclusive operation.

```
ember-template-lint app/**/*.js - --filename foobar.js
```

Will not be supported. If that is intentional then it might be worth redoing the arguments to be explicit about this. For example not relying on `-` on its own. Maybe this to turn on STDIN **mode**

```
ember-template-lint --read-from-stdin --filename foobar.js
```

Thoughts?

Reply:
> This will strait up block any globbing. 

Yes, that is intentional from my perspective. Using both stdin and filename globs together seems very bizarre. Either we lint stdin or we lint globs, not both.